.travis.yml

```diff
@@ -3,5 +3,8 @@ language: ruby
 cache: bundler
 script:
   - docker-compose -f travis-docker-compose.yml run web tox
+  - docker-compose -f travis-docker-compose.yml run watch npm install
```

Comment:
We had set NODE_ENV=development in `tox.ini` which overrides anything set in the docker-compose file. But it doesn't look like there was a good reason for doing that so it's best to leave it out